rbc/heavydb/text_encoding_none.py

```diff
@@ -47,6 +48,25 @@ def match(self, other):
             return 2
 
 
+class TextEncodingNonePointer(ArrayPointer):
+    def copy(self, context, builder, val, retptr):
+        from .buffer import memalloc
+        ptr_type = self.dtype.members[0]
+        element_size = int64_t(ptr_type.dtype.bitwidth // 8)
+
+        struct_load = builder.load(val)
+        src = builder.extract_value(struct_load, 0, name='text_buff_ptr')
+        element_count = builder.extract_value(struct_load, 1, name='text_size')
```

Comment:
If `element_count` is negative, we could use `strlen` to compute the actual `element_count`?

Reply:
See my comment above. `element_count` should never be negative.